node/pkg/publicrpc/publicrpcserver.go

```diff
@@ -75,6 +76,12 @@ func (s *PublicrpcServer) GetSignedVAA(ctx context.Context, req *publicrpcv1.Get
 	addr := vaa.Address{}
 	copy(addr[:], address)
 
+	// Chain input validation
+	chainId, err := vaa.ChainIDFromInt(uint16(req.MessageId.EmitterChain))
```

Comment:
Here we're validating if `uint16(req.MessageId.EmitterChain)` is valid, but then in L86 we're reading `vaa.ChainID(req.MessageId.EmitterChain.Number())`. I'm not sure if there is a difference between the two, but there could be and we should make sure that we're validating what we're actually using at the end.

Reply:
I'll change them to be consistent.

Reply:
made them consistent.